wgpu-core/src/command/compute.rs

```diff
@@ -459,7 +459,7 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
                     }
                 }
                 ComputeCommand::DispatchIndirect { buffer_id, offset } => {
-                    let scope = PassErrorScope::DispatchIndirect;
+                    let scope = PassErrorScope::Dispatch{indirect:true, pipeline:state.pipeline.last_state};
```

Comment:
did you run `cargo fmt` on this?

Reply:
I did, but I guess I noticed the dispatch stuff after that, oops.